.github/workflows/on_zigbee2mqtt_release.yaml

```
@@ -33,6 +33,7 @@ jobs:
           ZIGBEE2MQTT_VERSION=${{ github.event.inputs.zigbee2mqtt_version }}
           yq -i ".appVersion=\"${ZIGBEE2MQTT_VERSION}\"" charts/zigbee2mqtt/Chart.yaml
           yq -i ".version=\"${ZIGBEE2MQTT_VERSION}\"" charts/zigbee2mqtt/Chart.yaml
+          yq -i ".image.tag=\"${ZIGBEE2MQTT_VERSION}\"" charts/zigbee2mqtt/Chart.yaml
```

I think here you meant the values file. 
```suggestion
          yq -i ".image.tag=\"${ZIGBEE2MQTT_VERSION}\"" charts/zigbee2mqtt/values.yaml
```